t/sendreport/two-tier/email-both.t

```diff
@@ -0,0 +1,299 @@
+use FixMyStreet::Cobrand;
+use FixMyStreet::Script::Reports;
+use FixMyStreet::TestMech;
+use Test::Deep;
+use Test::MockModule;
+use Test::MockObject;
+
+my $mech = FixMyStreet::TestMech->new;
+
+my $user = $mech->create_user_ok('user@example.com');
+
+my $body_oxf = $mech->create_body_ok( 2237, 'Oxfordshire County Council',
+    {}, { cobrand => 'oxfordshire' } );
+my $body_cherwell
+    = $mech->create_body_ok(
+        2419,
+        'Cherwell District Council',
+        {},
+        { cobrand => 'oxfordshire' },
```

Comment:
This should not have a cobrand, because this body is not associated with any cobrand.